src/components/categorizedNotices/CategorizedNotices.tsx

```
@@ -1,31 +1,56 @@
+import React from "react";
 import { useTranslation } from "react-i18next";
+import useSWR from "swr";
+import axios from "axios";
+import Zabo from "../zabo/Zabo";
+import { ex_data } from "./example";
+import { Notice } from "@/types/interfaces";
+import SearchNoResult from '@/assets/icons/search-no-result.svg?react'
+import Loading from "../loading/Loading";
+import Error from "@/assets/error/Error";
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)